### DISCUSSION TITLE
我的小记 - 好记性不如烂笔头

### DISCUSSION THREAD

**Discussion by giscus** [xiaodongxier/wiki#9] [Announcements]
# 我的小记 - 好记性不如烂笔头

王永杰的个人网站,王永杰的个人wiki,小东西儿的个人网站,xiaodongxier.com,wangyongjie.cn,前端知识管理,个人wiki,个人知识管理

https://wiki.wangyongjie.cn/%20%E9%A6%96%E9%A1%B5/%E6%88%91%E7%9A%84%E5%B0%8F%E8%AE%B0/

**Comment by xiaodongxier** [comment]
20240906 豆豆放学说明天是《白露》节气，天气会变冷

**Comment by xiaodongxier** [comment]
全平台局域网同步的软件，
LANDrop
LocalSend

**Comment by xiaodongxier** [comment]
沉没成本不参与重大决策
在做出决策时，不能因为已经投入了大量时间、金钱或其他资源而坚持做出不合理的选择。
沉没成本是指已经发生且不可逆转的成本，无论做出什么决策，这部分成本都无法收回。
因此，在做决策时，应以当前和未来的利益为出发点，而不是过去已经发生的成本。
这样做可以避免持续投入资源而无法获得预期回报的情况。
延伸来看，在个人生活或商业领域，人们应该理智地评估已经投入的成本，并基于当前和未来的利益来做出决策，而不是被过去的投入所束缚。
这样才能更好地实现最大化的利益。

**Comment by xiaodongxier** [comment]
亲人的离去不是一场暴雨，而是此生漫长的潮湿，我永远困在这潮湿中，在每一个波澜不惊的日子里，掀起狂风暴雨！

**Reply by xiaodongxier** [reply]
在婚姻之前失去父母，真的会很孤独，这就是伴侣的意义

**Reply by xiaodongxier** [reply]
https://www.v2ex.com/t/1078509

**Comment by xiaodongxier** [comment]
专家说农村宅基地闲置:有村庄空置率43.8% 

网友经典评论：那不是闲置房，那是我们的故乡，是我们一年一度疗伤的港湾，是我们为生活不得不闲置的无可奈何！！！

**Comment by xiaodongxier** [comment]
法人人格混同
